oktapam/data_source_project_group.go

```diff
@@ -67,30 +66,25 @@ func dataSourceProjectGroup() *schema.Resource {
 func dataSourceProjectGroupFetch(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	c := m.(client.OktaPAMClient)
 
-	group := d.Get(attributes.GroupName).(string)
-	if group == "" {
-		return diag.Errorf("%s cannot be blank", attributes.GroupName)
-	}
-
 	project := d.Get(attributes.ProjectName).(string)
-	if project == "" {
-		return diag.Errorf("%s cannot be blank", attributes.ProjectName)
-	}
+	group := d.Get(attributes.GroupName).(string)
```

Comment:
I liked the projectName / groupName pattern you were using elsewhere.

Reply:
Fixed